nautobot/ui/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "nautobot",
   "version": "2.0.0-alpha.1",
   "private": true,
+  "proxy": "http://nautobot:8080",
```

Comment:
This feels like a strange place for this, though I'm definitely no expert here. What's the impact of this addition?

Reply:
This is what tells node to proxy any unknown routes to the backend nautobot container (so we can remove nginx)